app/indexers/default_object_rights_indexer.rb

```diff
@@ -13,7 +13,9 @@ def to_solr
 
     {
       'use_statement_ssim' => use_statement,
-      'copyright_ssim' => copyright
+      'copyright_ssim' => copyright,
+      'rights_descriptions_ssim' => 'Dark',
+      'default_rights_descriptions_ssim' => RightsDescriptionBuilder.build(cocina)
```

Comment:
Shouldn't the parameter be `cocina.defaultAccess` ?

Reply:
No because items need to dig into places other than the access node (they go into structural), which is why I left the paramter as the root of the cocina model and then deciding where to find the rights based on the type (APO vs item).

see for example: https://github.com/sul-dlss/dor_indexing_app/blob/index-rights-description-for-apos/app/services/rights_description_builder.rb#L77-L82